hscript/Parser.hx

```diff
@@ -821,6 +822,18 @@ class Parser {
 				))
 			]),pmin(e1));
 			return parseExprNext(e);
+		case TQuestionQuestion:
+			var e2 = parseExpr();
+			var tmp = "__a_" + (uid++);
+			var e = mk(EBlock([
+				mk(EVar(tmp, null, e1), pmin(e1), pmax(e1)),
+				mk(ETernary(
+					mk(EBinop("==", mk(EIdent(tmp),pmin(e1),pmax(e1)), mk(EIdent("null"),pmin(e1),pmax(e1)))),
+					e2,
+					e1
```

Comment:
Shouldn't this be `tmp` to avoid code duplication?

Reply:
Yeah my bad

Reply:
This also means that a relevant test is missing.

Reply:
i just made one and after the change runs successfully, but maybe pushing it would be a lot of code just for it that now works great. I had to make a new class with a integer variable with a getter function that increases everytime you access it